graylog2-web-interface/src/components/navigation/SystemMenu.jsx

```diff
@@ -146,16 +146,16 @@ const SystemMenu = ({ location }) => {
           <NavigationLink path={Routes.SYSTEM.AUTHENTICATION.BACKENDS.ACTIVE} description="Authentication" />
         </IfPermitted>
       </HideOnCloud>
-      <IfPermitted permissions={['dashboards:create', 'inputs:create', 'streams:create']}>
+      <IfPermitted permissions={['contentpack:read']}>
         <NavigationLink path={Routes.SYSTEM.CONTENTPACKS.LIST} description="Content Packs" />
       </IfPermitted>
-      <IfPermitted permissions={['inputs:read']}>
+      <IfPermitted permissions={['grok_pattern:read']}>
         <NavigationLink path={Routes.SYSTEM.GROKPATTERNS} description="Grok Patterns" />
       </IfPermitted>
-      <IfPermitted permissions={['inputs:edit']}>
+      <IfPermitted permissions={['lookuptables:read']}>
         <NavigationLink path={Routes.SYSTEM.LOOKUPTABLES.OVERVIEW} description="Lookup Tables" />
       </IfPermitted>
-      <IfPermitted permissions={['inputs:create']}>
+      <IfPermitted permissions={['pipeline:read', 'pipeline_connection:read']}>
```

Comment:
Hmm, we are changing the permissions from "write" to "read" here. Is that really what we want? Do all the pages work for users that only have `*:read` permissions to the subsystems?

Reply:
@bernd I originally went with `*:write`, but I tested these out and users with only `*:read` permissions can navigate to the subsystem pages and view the contents but can't make any changes. So I went with the minimal permissions to navigate to the page.

If you think that's not a realistic use case or too permissive I can definitely update the permissions to `*:write` here.

Reply:
@ryan-carroll-graylog Okay, thanks for the explanation.

Did you test with a user with only the "Reader" role? Does anything in the menu change for these users with this PR applied?

Reply:
@bernd No problem. There is one change for a user with only the "Reader" role, because it seems to be tied to `inputs:read`: They can no longer see Grok Patterns. Otherwise they are still unable to see the other updated subsystems in the dropdown.

Reply:
Alright, sounds good! Thank you! :+1:

Reply:
Thanks for taking a look!